describe/test_describe.py

```diff
@@ -40,3 +52,46 @@ def test_run_local():
             "violin.html",
         }
     )
+
+
+def test_sanity_local():
```

Comment:
every function / class / global variable should have a doc string, you can look at `onnx_utils` for exmaple.